README.md

```diff
@@ -58,7 +58,7 @@ To accomodate different timezones, we schedule community calls at different time
 
 | Asset | Link        |
 |:-----------|:------------|
-| 🔗 Meeting Link & Password | **Zoom Call:** https://us02web.zoom.us/j/85305980190<br>**Password:** eWRhSklVTjJjSnhTaURDcFZaU2ZzQT09
+| 🔗 Meeting Link & Password | **Zoom Call:** https://zoom.us/j/92337656846?pwd=R0dHRE9NRXlLalpXRXNhblBsOG1LUT09<br>**Password:** 815623
```

Comment:
Please, don't put the password in the URL since there are bots that scrape for those and auto-join the meetings. We had situations like that in the past.